processor/probabilisticsamplerprocessor/config.go

```diff
@@ -15,26 +15,76 @@
 package probabilisticsamplerprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/probabilisticsamplerprocessor"
 
 import (
+	"fmt"
+
 	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/model/pdata"
 )
 
-// Config has the configuration guiding the trace sampler processor.
+var severityTextToNum = map[string]pdata.SeverityNumber{
+	"default": pdata.SeverityNumberUNDEFINED,
```

Comment:
Don't we have a map like this elsewhere already? I feel like it would better belong elsewhere.

Reply:
I looked and looked - nope. There is some stuff in opentelemetry-log-collection, but not explicitly a text to severity number.